addons/account/models/chart_template.py

```diff
@@ -301,12 +301,20 @@ def _pre_reload_data(self, company, template_data, data):
         current_taxes = self.env['account.tax'].with_context(active_test=False).search([
             *self.env['account.tax']._check_company_domain(company),
         ])
+
+        current_fiscal_positions =  self.env['account.fiscal.position'].with_context(active_test=False).search([
+            *self.env['account.fiscal.position']._check_company_domain(company),
+        ])
         unique_tax_name_key = lambda t: (t.name, t.type_tax_use, t.tax_scope, t.company_id)
         unique_tax_name_keys = set(current_taxes.mapped(unique_tax_name_key))
         xmlid2tax = {
             xml_id.split('.')[1].split('_', maxsplit=1)[1]: self.env['account.tax'].browse(record)
             for record, xml_id in current_taxes.get_external_id().items() if xml_id.startswith('account.')
         }
+        xmlid2fiscal_position= {
```

Comment:
name of the variable is not that great, i would maybe call it account_fiscal_positions or something like that ?

Reply:
I know that the name is not god but I have followed the same convention of  xmlid2tax that was already in the code.

No problem I will do the modifications

Reply:
ok for me :smile: